rules/crowdstrike_rules/crowdstrike_base64_encoded_args.py

```diff
@@ -25,12 +25,15 @@ def rule(event):
 
     # Split arguments from process path
     command_line_args = event.udm("cmd")
-    command_line_args = command_line_args.replace('"', "")
-    command_line_args = command_line_args.replace("'", "")
-    command_line_args = command_line_args.split(" ")[1:]
+    command_line_args = command_line_args.replace('"', " ")
+    command_line_args = command_line_args.replace("'", " ")
+    command_line_args = command_line_args.replace("=", " ")
+    command_line_args = command_line_args.split(" ")
```

Comment:
Is there any reason we're including the root command in `command_line_args`? Previously we had `command_line_args.split(" ")[1:]` and I'm not sure why the change

Reply:
good catch, readded